app/installer.sh

```diff
@@ -107,6 +110,53 @@ if [ "$OS" = "Linux" ]; then
     printf "INFO\tDistribution: %s\n" "$DISTRO"
 fi
 
+# Sets `VERSION` to the git tag representing the latest preview version.
+set_preview_version() {
+  if [ ! -x `which gh` ]; then
+    echo "You must have github's gh client installed to install a preview version"
+    exit 1
+  fi
+
+  IFS=/ read -r owner repo << EOF
+${REPO}
+EOF
```

Comment:
I guess: REPO_PREFIX is the repo here.

Reply:
I’m not following. What do you mean by this?

Reply:
When you do: `REPO_PREFIX=$(echo $REPO | sed 's|.*/||')` you end up with REPO_PREFIX that is equal to you `repo` variable extracted here. Maybe it's good enough to have just once

Reply:
Ah, now I get it. Yes, indeed. I’ve replaced this `read` usage altogether now and construct a new `REPO_ORG` variable in the same `sed` style as `REPO_PREFIX`.